tox.ini

```diff
@@ -30,6 +30,17 @@ commands =
     coverage xml -i
     coverage report
 
+[testenv:requirements]
```

Comment:
Please, add the requirements in the tox envlist (line 2 of this file)

```ini
[tox]
envlist = py39,py310,requirements,lint,test
```

Reply:
Done.